Formula/stone-soup.rb

```diff
@@ -53,6 +53,12 @@ def install
         args << "SOUND=y"
       end
 
+      if build.with? "lua"
```

Comment:
And `if build.with? "lua@5.1"` down here too. That should be the last change necessary in terms of correct style/usage. After that this should be fine for a maintainer to review for acceptability in general.